keyboards/fractal/fractal.h

```diff
@@ -0,0 +1,37 @@
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+
+#define LAYOUT_preonic_grid( \
```

Comment:
@noroadsleft or @mechmerlin, is this what we want here?

Reply:
Should be changed to `LAYOUT_ortho_5x12` to support the community layout feature.

I'll add some review comments on the relevant files.